src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+
+
+public class Duke {
+    public static final String horizontalLine = "____________________________________________________________\n";
+    public static final ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public static void main(String[] args) throws DukeException {
+        printGreetings();
+        execute();
+
+    }
```

Comment:
If your `main` throws DukeException, OS will handle it by crashing the app. Maybe you want to catch the exception within `main` or any function called in main to handle the exception, so your program doesn't crash?